spec/unit/provider/package/apt_spec.rb

```diff
@@ -289,5 +289,18 @@
 
       provider.install
     end
+
+    it "should install using the source attribute if present" do
+      resource[:ensure] = :installed
+      resource[:source] = '/my/local/package/file'
+
+      expect(provider).to receive(:aptget) do |*command|
+        expect(command[-1]).to eq(resource[:source])
+        expect(command[-2]).to eq(:install)
+      end
```

Comment:
Alternatively, i think this could work:
```ruby
expect(provider).to receive(:aptget).with(any_args, :install, resource[:source])
```

Reply:
Either I could write like you proposed or move the test near another test handling similar case: https://github.com/puppetlabs/puppet/blob/5b13c441da896e04da8bf064160efa690512f6f2/spec/unit/provider/package/apt_spec.rb#L178-L187

Reply:
I'm fine with keeping it as it is for consistency